src/reducers/api.tsx

```diff
@@ -0,0 +1,34 @@
+import { Reducer } from 'redux';
+import { ActionType, createAction, getType } from 'typesafe-actions';
+
+type SetAuthTokenPayload = {
+  authToken: string;
```

Comment:
`string | null`, right? I'm thinking of when logging out will need to reset `authToken`.